src/fund_my_watcard/main.py

```diff
@@ -128,5 +135,58 @@ def main():
         if query_yes_no(WILL_RESET_YOUR_CONFIG_FILE_WARNING, "no"):
             reset_config_file()
 
+    elif args.valid:
+        if os.path.isfile(CONFIG_FILE_PATH):
+            with open(CONFIG_FILE_PATH) as f:
+                _config = json.load(f)
+        else:
+            report_error(CAN_NOT_FIND_CONFIG_FILE)
+
+        if _config["encrypted"] == "True":
+            report_warning(CONFIG_FILE_HAS_BEEN_ENCRYPTED)
```

Comment:
When the user's config file is encrypted, report an error. Since we don't want the program to check an encrypted file.